From e73cf28a72a02ce4f56471829a97c70dc4831a84 Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Wed, 31 Jul 2013 08:04:20 +0200
Subject: qxl: set revision to 1 for rhel6.0.0

Message-id: <1375201922-6794-9-git-send-email-armbru@redhat.com>
Patchwork-id: 52818
O-Subject: [RHEL-7 PATCH v3 08/15] qxl: set revision to 1 for rhel6.0.0
Bugzilla: 983991
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Bandan Das <bsd@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>

From: Gerd Hoffmann <kraxel@redhat.com>

bugzilla: #893344 - info qtree mismatch
upstream: No (rhel6.0.0 spice code base predates upstream merge,
              so this compat property isn't relevant there).

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
Signed-off-by: Michal Novotny <minovotn@redhat.com>
(cherry picked from commit 69f07de9ed3678198578fe036da202cbbca3f312)

Conflicts:
        hw/pc.c

Picked from qemu-kvm-rhel6.

Signed-off-by: Markus Armbruster <armbru@redhat.com>

diff --git a/hw/i386/pc_piix.c b/hw/i386/pc_piix.c
index 9e96b79..57877e3 100644
--- a/hw/i386/pc_piix.c
+++ b/hw/i386/pc_piix.c
@@ -870,11 +870,11 @@ static QEMUMachine pc_machine_rhel700 = {
     },{\
         .driver   = "qxl",\
         .property = "revision",\
-        .value    = stringify(3),\
+        .value    = stringify(1),\
     },{\
         .driver   = "qxl-vga",\
         .property = "revision",\
-        .value    = stringify(3),\
+        .value    = stringify(1),\
     },{\
         .driver   = "VGA",\
         .property = "mmio",\
